cmd/execution/main.go

```diff
@@ -224,6 +224,11 @@ func main() {
 			chunkDataPacks := storage.NewChunkDataPacks(node.DB)
 			stateCommitments := storage.NewCommits(node.Metrics.Cache, node.DB)
 
+			// Needed for gRPC server, make sure to assign to main scoped vars
+			events = storage.NewEvents(node.Metrics.Cache, node.DB)
+			serviceEvents := storage.NewServiceEvents(node.Metrics.Cache, node.DB)
```

Comment:
should this also be assigned to the main scoped var?

Reply:
good catch 👍